js/config.js

```diff
@@ -1,13 +1,14 @@
 module.exports = {
   output: './test/generic',
   ext: '.spec.js',
-  emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/js/emitter.js',
+  emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/universal-sdk/js/emitter.js',
```

Comment:
@Artem-applitools after merging to `master` these changes should be reverted, right?

Reply:
@amitzur Yes, after we move all sdks to the universal, we merge this branch to master and those changes should be reverted